### ISSUE TITLE
Remaining issues for 1.0 release

### ISSUE THREAD

**Issue by sjackso** [lcbb/athena#3]
As of v0.2.1 the GUI is feature-complete, but there are a few things that need to happen before we can move beyond beta status.

- [x] Scaffold files -- for now we've decided that Athena can just ship with an m13 default
    - [x] Scaffold import still needs UI, this should be quick to add.

- [x] The repository README needs to be made ready for reading by newcomers.

- [x] The LICENSE information needs to be updated to attribute code that has been brought in from other sources.  (Specifically: Qt example code for QML and shader files, and the earcut code for triangulation.)

- [x] The build files should be modified to include end-user README and LICENSE files in the distribution package.

- [x] The built-in 2d and 3d example files should perhaps be slimmed down, spelling errors corrected, and any attribution given.  (Do we have a right to distribute some / all / none of these?)

The following items are NICE TO HAVES for version 1.0, but we could live without them:

* The build files should apply version information automatically to the binary and the README, so that Athena as a self-contained artifact has accurate version information for users to report bugs.

* Athena should report the progress of sequence tools when they are running, ideally in a way that does not halt the program while they run.

 * ~~If the above can be done, then the text console can be hidden at launch time, leaving Athena as a truly one-window program.~~  This had to be done to allow Athena to become Athena.app on OSX.  Tools are dumping their progress into an in-process log window instead of a separate console window now, but they still halt the program while processing.

* Make color selection windows be non-modal (i.e. make them not halt the rest of the GUI).
